<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Step2</class>
 <widget class="QMainWindow" name="Step2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DB-MigrationTool - Step 2 - Compare and Migrate</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/Images/mainIcon</normaloff>:/Images/mainIcon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="cmbFirstDB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbSecondDB">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>70</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="TabView" name="tabFirstDB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::CopyAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="db1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Database 1</string>
    </property>
    <property name="buddy">
     <cstring>cmbFirstDB</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="db2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Database 2</string>
    </property>
    <property name="buddy">
     <cstring>cmbSecondDB</cstring>
    </property>
   </widget>
   <widget class="TabView" name="tabSecondDB">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::CopyAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Please select working databases.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="cmdPrev">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>172</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Previous step</string>
    </property>
    <property name="icon">
     <iconset resource="rc.qrc">
      <normaloff>:/Images/arrowB</normaloff>:/Images/arrowB</iconset>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkCopyWithoutContents">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>10</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drag without contents</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>35</y>
      <width>641</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompare">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Compare tables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompareByContent">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>110</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Compare by content</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>597</x>
      <y>410</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TabView</class>
   <extends>QTableView</extends>
   <header location="global">tabview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
